Dockerfile

```diff
@@ -0,0 +1,28 @@
+# Defining ruby version
+FROM ruby:2.6.6-alpine
+
+# Copy app and set working dir
+COPY . /application
+WORKDIR /application
+
+# Prerequisites for gems install
+RUN rm -rf /application/Gemfile.lock
+RUN apk add build-base \
+            npm \
+            tzdata \
+            git
+
+# Install bundler and gems
+RUN gem install bundler:2.1.4
+RUN bundle install --without="development"
+
+# Set environment variables and expose the running port
+ENV RELATIVE_URL_ROOT="/"
+ENV RAILS_ENV="production"
```

Comment:
Should also be passed in via the container config, although `production` is the right default